### ISSUE TITLE
"Fatal Error" orrurs during LAN

### ISSUE THREAD

**Issue by Mailyn001** [WildBamaBoy/minecraft-comes-alive#419]
Me and my sister are tyring to play a LAN game. But once one player likes to join the server, he immediately gets the message that a fatal error was caused (This sometimes happens to the host, too). One can be in-game with the other player for about 1-10 seconds and then this happenes.
Singleplayer works.

We both already re-installed our Minecraft. MCA and RadixCore (with Forge) are the only mods running for both of us, and we made sure to download the latest versions. Therefore we have the same versions and also the same settings.
We already shat down the firewall. And when we play without mods everything works fine.

Any suggestions?

**Comment by Hernan0708** [comment]
are you playing on 1.8? because 1.8 v5.0.6.1 is not currenty capable with multiplayer.

btw post your crash report for proof.

**Comment by Hernan0708** [comment]
or is it 5.0.6..... idk

**Comment by Mailyn001** [comment]
Wow, thanks for the fast answer!
Yes, we are playing with the 1.8 version. Wich one works for multiplayer?

CRASH REPORT SERVER

---- Minecraft Crash Report ----
// You're mean.

Time: 20.06.15 21:09
Description: Exception in server tick loop

java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1429)
	at java.util.HashMap$KeyIterator.next(HashMap.java:1453)
	at com.google.common.collect.AbstractMapBasedMultimap$WrappedCollection$WrappedIterator.next(AbstractMapBasedMultimap.java:486)
	at net.minecraft.util.ClassInheritanceMultiMap$2.computeNext(SourceFile:90)
	at com.google.common.collect.AbstractIterator.tryToComputeNext(AbstractIterator.java:143)
	at com.google.common.collect.AbstractIterator.hasNext(AbstractIterator.java:138)
	at net.minecraft.world.World.func_175650_b(World.java:3126)
	at net.minecraft.world.chunk.Chunk.func_76631_c(Chunk.java:952)
	at net.minecraftforge.common.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:46)
	at net.minecraftforge.common.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:12)
	at net.minecraftforge.common.util.AsynchronousExecutor.skipQueue(AsynchronousExecutor.java:344)
	at net.minecraftforge.common.util.AsynchronousExecutor.getSkipQueue(AsynchronousExecutor.java:302)
	at net.minecraftforge.common.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:12)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:117)
	at net.minecraft.world.gen.ChunkProviderServer.func_73158_c(ChunkProviderServer.java:92)
	at net.minecraft.server.MinecraftServer.func_71222_d(MinecraftServer.java:302)
	at net.minecraft.server.integrated.IntegratedServer.func_71247_a(IntegratedServer.java:112)
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:126)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:438)
	at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.8
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 443348216 bytes (422 MB) / 930877440 bytes (887 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 15, tcache: 0, allocated: 13, tallocated: 95
	FML: MCP v9.10 FML v8.0.99.99 Minecraft Forge 11.14.3.1450 5 mods loaded, 5 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAAAAAAAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAAAAAAAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.8-11.14.3.1450.jar) 
	UCHIJAAAAAAAAAAAA	Forge{11.14.3.1450} [Minecraft Forge] (forge-1.8-11.14.3.1450.jar) 
	UCHIJAAAAAAAAAAAA	RadixCore{2.0.3-1.8} [RadixCore] (RadixCore-2.0.3-1.8-universal.jar) 
	UCHIJAAAAAAAAAAAA	MCA{5.0.6.1-1.8} [Minecraft Comes Alive] (MCA-5.0.6.1-1.8-universal.jar) 
	Loaded coremods (and transformers): 
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Player Count: 0 / 8; []
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'



CRASH REPORT CLIENT

---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 20.06.15 22:51
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at mca.core.forge.EventHooksFML.clientTickEventHandler(EventHooksFML.java:123)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_14_EventHooksFML_clientTickEventHandler_ClientTickEvent.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:55)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:138)
	at net.minecraftforge.fml.common.FMLCommonHandler.onPostClientTick(FMLCommonHandler.java:371)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2152)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1028)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:345)
	at net.minecraft.client.main.Main.main(SourceFile:120)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.8
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 584121464 bytes (557 MB) / 930287616 bytes (887 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 94
	FML: MCP v9.10 FML v8.0.99.99 Minecraft Forge 11.14.3.1450 5 mods loaded, 5 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAAAAAAAAAAAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAAAAAAAAAAAAAA	FML{8.0.99.99} [Forge Mod Loader] (forge-1.8-11.14.3.1450.jar) 
	UCHIJAAAAAAAAAAAAAAAA	Forge{11.14.3.1450} [Minecraft Forge] (forge-1.8-11.14.3.1450.jar) 
	UCHIJAAAAAAAAAAAAAAAA	RadixCore{2.0.3-1.8} [RadixCore] (RadixCore-2.0.3-1.8-universal.jar) 
	UCHIJAAAAAAAAAAAAAAAA	MCA{5.0.6.1-1.8} [Minecraft Comes Alive] (MCA-5.0.6.1-1.8-universal.jar) 
	Loaded coremods (and transformers): 
	GL info: ' Vendor: 'Intel' Version: '4.2.0 - Build 10.18.10.3277' Renderer: 'Intel(R) HD Graphics 4600'
	Launched Version: 1.8-Forge11.14.3.1450
	LWJGL: 2.9.1
	OpenGL: Intel(R) HD Graphics 4600 GL version 4.2.0 - Build 10.18.10.3277, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: [PureBDcraft 128x MC18.zip]
	Current Language: Deutsch (Deutschland)
	Profiler Position: N/A (disabled)

**Comment by Hernan0708** [comment]
only MCA 1.7.10 v5.0.6 or older works with servers right now

**Comment by Hernan0708** [comment]
try playing or switch to a 1.7.10 server until the bug is fixed.

**Comment by Mailyn001** [comment]
Oh wow think you! Is there any website I can keep up to date when the bug is fixed?

**Comment by Hernan0708** [comment]
for what? forge or.... mca??

**Comment by Mailyn001** [comment]
Ahm... The cause of the troubles is MCA, right? So for MCA :)

**Comment by Hernan0708** [comment]
To keep up with MCA. check the bug tracker.

i check it a lot of times during the week. so i can see a percentage of progress until Wildbamaboy is complete and releases MCA 5.1

Here is the link: https://github.com/WildBamaBoy/minecraft-comes-alive/milestones

**Comment by Mailyn001** [comment]
Thank you my friend!! :D

**Comment by WildBamaBoy** [comment]
Determined not an MCA issue.

**Comment by TheProjectShaman** [comment]
Hi there! 

I'm setting up a Sponge Server (so I can use both plugins and mods at the same time in 1.8), and noticed that MCA (which is awesome btw!) was crashing when I used it in multiplayer LAN (using a Technic custom modpack I created). Unfortunately I am forced to use Forge 1487 (for 1.8) so I can get client-side shaders to work (OptiFine_1.8.0_HD_U_D5.jar AND ShadersMod-v2.4.12mc1.8.jar). No other mods were crashing, and I even had it on its own (just MCA-1.8-5.0.7.1-universal.jar AND RadixCore-1.8-2.0.5-universal.jar) but it crashed then as well.  One of the error lines in the console said: "There was a critical exception handling a packet on channel RadixCore". Do I need to change the version of Forge, or is this still an open multiplayer issue for 1.8? Or perhaps is this a RadixCore issue?

Either way I noticed this bug was no longer an open issue for the upcoming version of MCA 5.1 and wanted to ensure that this 1.8 multiplayer issue was still on the radar for fixing.

Those of us with 1.8 servers would love to be able to use MCA. 

Thanks in advance for all your amazing work! :)
